docs/api-reference/mapbox/mapbox-layer.md

```diff
@@ -101,6 +101,7 @@ Parameters:
 - `props` (Object)
   + `props.id` (String) - an unique id is required for each layer.
   + `props.deck` (`Deck`, optional) - a `Deck` instance that controls the rendering of this layer. If provided, the layer will be looked up from its layer stack by `id` at render time, and all other props are ignored.
+  + `props.viewId` (String, optional) - Provide the `id` of a `MapView` instance to render the Map to (e.g. if using multiple `views` on a `Deck` instance). If it is not provided, the Map will be rendered on a single viewport. If a viewId is not found, nothing will be done to the viewports.
```

Comment:
Required if this layer should be rendered into multiple views. Specify the id of a view that corresponds to the base map.

- If not provided, the layer will only be drawn into the base map's viewport.
- If provided, the viewport with this id will be replace to match the base map.

Reply:
We decided against this in the end, resolving.